api/app/src/default-error-handler.ts

```diff
@@ -43,7 +44,25 @@ export const errorHandler: ErrorRequestHandler = (err, req, res, next) => {
     return res.contentType("json").status(err.status).send(metriportResponseBody(err));
   }
   if (err instanceof ZodError) {
-    return res.contentType("json").status(status.BAD_REQUEST).send(zodResponseBody(err));
+    return res.contentType("json").status(httpStatus.BAD_REQUEST).send(zodResponseBody(err));
+  }
+  if (err instanceof OperationOutcomeError) {
+    const status = httpStatus.INTERNAL_SERVER_ERROR;
+    if (req.path.includes("fhir/R4")) {
+      return res.contentType("json").status(status).send(err.outcome);
```

Comment:
do we need to update the docs accordingly? this is mentioned in the issue #682

Reply:
I suggest we leave this to another PR (I'll add a note on the ticket), because this PR's version is exposing any FHIR originated error that's not related to `/fhir/R4` to RFC7807 - so this is an improvement.

`/fhir/R4` is not being documented yet, so when we do it, then we mention this difference.

What do you think?

Reply:
Yup agreed - as long as it's a conscious decision and we don't forget about it